scipy/fftpack/tests/test_basic.py

```diff
@@ -127,7 +127,7 @@ def direct_irdft(x):
     return direct_idft(x1).real
 
 
-class _TestFFTBase(TestCase):
+class _TestFFTBase(object):
     def setUp(self):
```

Comment:
Does the `setUp` method (and other unittest hook methods) get still called? (From a quick local test, it seems not.)